tests/test_expand_expr.c

```diff
@@ -18,7 +18,7 @@ extern t_hashtable	*g_symtab;
 extern t_info		g_info;
 extern int			g_err_count;
 
-int	test_wrapper(char *testname, char *test, char *expect)
+static int	test_wrapper(char *testname, char *test, char *expect)
```

Comment:
I had the same problem of redefinition :D I recently used test_ft_name() so it's clear which function is tested

Reply:
hmmm? but isn't that a general function which is used by all tests?? show me what you mean

Reply:
I think you made this one static (in test_expand), since you also called yours test_wrapper? 
In builtin test I called mine test_wrapper at first, got a redefinition error and then just called it exec_ft_echo() - not "test_.." as I wrote. 
So a function which executes a function to be tested has the form exec_ft_name(). For example if you want to test subfunctions of your module. But in the end - I don't think it matters :D

Reply:
Gotta see your code to see what you mean. As the static solutions works just fine, let's leave it like that

Reply:
Yeah they work! I just found it funny that I didn't think of that (just make it static :D)